bosk-testing/src/main/java/io/vena/bosk/drivers/DriverConformanceTest.java

```diff
@@ -242,7 +243,7 @@ void testOptional() throws InvalidTypeException {
 		driver.submitReplacement(ref, TestValues.blank().withString("changed"));
 		assertCorrectBoskContents();
 
-		assertThrows(NullPointerException.class, ()->driver.submitReplacement(ref, null));
+		assertThrows(NullPointerException.class, ()->driver.submitReplacement(ref, Optional.of(null)));
```

Comment:
This is no longer testing the same thing. `Optional.of` throws `NullPointerException` itself, so this isn't testing the bosk library anymore.

If you're getting an error from the compiler, you might want to replace `null` with `(TestValues)null`.

Reply:
Hm. This sitll doesn't seem to satisfy the compiler...